Bio/Phylo/Applications/_Fasttree.py

```diff
@@ -44,15 +44,23 @@ class FastTreeCommandline(AbstractCommandline):
 
     Examples
     --------
-    >>> import _Fasttree
-    >>> fasttree_exe = r"C:\FasttreeWin32\fasttree.exe"
-    >>> cmd = _Fasttree.FastTreeCommandline(fasttree_exe,
-    ...                                     input=r'C:\Input\ExampleAlignment.fsa',
-    ...                                     out=r'C:\Output\ExampleTree.tree')
-    >>> print(cmd)
-    >>> out, err = cmd()
-    >>> print(out)
-    >>> print(err)
+    >>> import Bio.Phylo.Applications as BPA
+    >>> import os
+    >>> import tempfile
+    >>> # If FastTree executable is on your home directory
+    >>> home_path = os.path.expanduser('~')
+    >>> fasttree_path = os.path.join(home_path,'FastTree')
+    >>> input_file = "./TreeConstruction/msa.phy"
+    >>> # Create a temporal file for the output
+    >>> f_descriptor, output_file = tempfile.mkstemp()
+    >>> # We do not need the file descriptor, clear it
+    >>> os.close(f_descriptor)
+    >>> cmd = BPA.FastTreeCommandline(fasttree_path, input=input_file, out=output_file)
+    >>> o_str, e_str = cmd()
+    >>> print(open(output_file, 'r').read())
+    ((Delta:0.24771,Epsilon:0.00054)0.999:0.71510,Beta:0.00054,(Gamma:0.34881,Alpha:0.29449)0.553:0.06991);
+    <BLANKLINE>
+    >>> os.remove(output_file)
```

Comment:
This can't be a doctest - we cannot assume that ``FastTree`` is on the $PATH, it may not even be installed.

Just leave the example is is, but change from ">>> " to four space indentation as an RST literal block. You may need to insert a line like this above with the trailing double colon:

```
This is an example on Windows::
```

Reply:
You can also use `>>`, this looks more similar to a Python command line but wouldn't invoke it as doctest.
I did so for other docstrings/doctests recently.

Reply:
Really? I missed that change and would have spoken out against it. Having a non-standard string like that is to me confusing, and being practical prevents simple copy-and-paste into an editor.

Note some editors (IPython IIRC) are aware of the standard Python ">>>" convention and will remove it on pasting in code.

Reply:
See d311208a530cf2525159d3cb76aeb1fefb8f243f